sktime/classification/deep_learning/esn.py

```diff
@@ -0,0 +1,159 @@
+__author__ = ["sharma-kshitij-ks"]
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+import numpy as np
+from scipy.sparse import random
+from sklearn.linear_model import RidgeClassifier
+from sktime.utils.validation import check_array
+from sktime.utils.validation.series import check_series
+from sktime.classification.base import BaseClassifier
+
+class EchoStateNetwork(BaseClassifier):
+    """Echo State Network (ESN) Classifier.
+
+    Overview:

```

Comment:
this should refer to the key reference, optimally how parameters of this implementation map to those in the paper.

You can number your references by using `[1]` etc in restructured text.